test/unit/js/flicking.test.js

```diff
@@ -15,22 +14,37 @@ function $getTransformValue(el, match) {
 }
 
 // create Flicking instance
-function create(el, option, handler) {
-	var customEvt = {
-		beforeFlickStart: handler,
-		flick : handler,
-		flickEnd : handler,
-		beforeRestore : handler,
-		restore : handler
-	};
+function create(el, option, customEvt) {
+	el = typeof el === "string" ? $(el)[0] : el;
+	el.eventFired = [];
+	el.eventDirection = [];
+	customEvt = customEvt || {};
+	var handler;
+
+	if ($.isPlainObject(customEvt)) {
+		handler = function(e) {
+			var type = e.eventType;
+			var fired = el.eventFired;
+			var direction = el.eventDirection;
 
-	if ($.isPlainObject(handler)) {
-		customEvt = handler;
-	} else {
-		handler = handler || $.noop;
+			if (fired.length === 0 || fired[fired.length - 1] !== type) {
+				fired.push(type);
+				direction.push(e.direction);
+			}
+		};
+	} else if(typeof customEvt === "function") {
+		handler = customEvt;
+		customEvt = {};
 	}
 
-	return new eg.Flicking(el, option || {}).on(customEvt);
+	return new eg.Flicking(el, option || {}).on(
+		$.extend({
+			beforeFlickStart: handler,
+			flick: handler,
+			flickEnd: handler,
+			beforeRestore: handler,
+			restore: handler
+		}, customEvt));
```

Comment:
How about manage instance in create method?
so, we will not miss releasing resources.